src/alks.js

```diff
@@ -223,7 +223,7 @@ class alks {
     * })
     */
   getAllAWSRoleTypes(props) {
-    const url = props.getDynamicValues ? 'allAwsRoleTypes' : 'allAwsRoleTypes?getDynamicValues=true'
```

Comment:
I'll take "Why ternary operators can obscure logic for 1000, Alex".  😄

Reply:
To be fair my dumb ass probably would've written 
```javascript
let url;
if (props.getDynamicValues) {
  url = 'allAwsRoleTypes'
} else {
  url = 'allAwsRoleTypes?getDynamicValues=true'
}
```
and gotten it backwards even if I didn't use the ternary 🤦